xclim/indices/_multivariate.py

```diff
@@ -717,6 +719,57 @@ def func(x, axis):
     return to_agg_units(out, tas, "count")
 
 
+@declare_units(
+    pr="[precipitation]",
+    tas="[temperature]",
+    pr_thresh="[precipitation]",
+    tas_thresh="[temperature]",
+)
+def high_precip_low_temp(
+    pr: xarray.DataArray,
+    tas: xarray.DataArray,
+    pr_thresh: str = "0.4 mm",
+    tas_thresh: str = "-0.2 C",
+    freq: str = "YS",
+) -> xarray.DataArray:  # noqa: D401
+    """Number of days with precipitation above threshold and temperature below threshold.
+
+    Number of days where precipitation is greater or equal to some threshold, and temperatures are colder than some
+    threshold. This can be used for example to identify days with the potential for freezing rain or icing conditions.
+
+    Parameters
+    ----------
+    pr : xarray.DataArray
+      Mean daily precipitation flux.
+    tas : xarray.DataArray
+      Daily mean, minimum or maximum temperature.
+    pr_thresh : str
+      Precipitation threshold to exceed.
+    tas_thresh : str
+      Temperature threshold not to exceed.
+    freq : str
+      Resampling frequency.
+
+    Returns
+    -------
+    xarray.DataArray, [time]
+      Count of days with high precipitation and low temperatures.
+
+    Example
+    -------
+    To compute the number of days with intense rainfall while minimum temperatures dip below -0.2C:
+    >>> pr = xr.open_dataset(path_to_pr_file).pr
+    `   >>> tasmin = x.open_dataset(path_to_tasman_file).tasmin
```

Comment:
?

Reply:
Ach saperlipopette.
There was a bug(?) where black would move the whole docstring 4 spaces to the right, making the build crash. Looks like I missed that line...